pkg/settings/migration.go

```diff
@@ -2,18 +2,22 @@ package settings
 
 // Environment variables.
 const (
-	FailureRollback = "FAILURE_ROLLBACK"
+	FailureRollback       = "FAILURE_ROLLBACK"
+	DisableImageMigration = "DISABLE_IMAGE_MIGRATION"
 )
 
 // Migration settings.
 //   FailureRollback: Should migration failure result in rollback of resources to source cluster?
+//   DisableImageMigration: Disable image migration from CAM
 type Migration struct {
-	FailureRollback bool
+	FailureRollback       bool
+	DisableImageMigration bool
 }
 
 // Load settings.
 func (r *Migration) Load() error {
 	r.FailureRollback = getEnvBool(FailureRollback, false)
+	r.DisableImageMigration = getEnvBool(DisableImageMigration, false)
```

Comment:
Going to temporarily set this to true so this is hardcoded into the image we are building for this test case.